clusterloader2/pkg/util/cluster.go

```diff
@@ -88,11 +104,16 @@ func isNodeSchedulable(node *corev1.Node) bool {
 	return !node.Spec.Unschedulable && nodeReady && networkReady
 }
 
-// Tests whether node doesn't have any taint with "NoSchedule" or "NoExecute" effect.
+// Tests whether node doesn't have any built-in taint with "NoSchedule" or "NoExecute" effect.
 func isNodeUntainted(node *corev1.Node) bool {
-	for i := range node.Spec.Taints {
-		if node.Spec.Taints[i].Effect == corev1.TaintEffectNoSchedule || node.Spec.Taints[i].Effect == corev1.TaintEffectNoExecute {
-			return false
+	for _, nodeTaint := range node.Spec.Taints {
+		if nodeTaint.Effect != corev1.TaintEffectNoSchedule && nodeTaint.Effect != corev1.TaintEffectNoExecute {
```

Comment:
@tosi3k wouldn't this always evaluate to false?

Reply:
Not really: https://github.com/kubernetes/api/blob/6c086a5e99d580d4cb60ffda862cb42683818f8f/core/v1/types.go#L2858.

Reply:
Right, my bad 👍  Missed the not equal